x/subscription/client/cli/tx.go

```diff
@@ -0,0 +1,274 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/spf13/cobra"
+
+	"github.com/crypto-org-chain/chain-main/v1/x/subscription/types"
+)
+
+// subscription flags
+const (
+	FlagTitle          = "title"
+	FlagDescription    = "description"
+	FlagPrice          = "price"
+	flagDurationSecs   = "duration-secs"
+	flagCronSpec       = "cron-spec"
+	flagPlanID         = "plan-id"
+	flagSubscriptionID = "subscription-id"
+	flagOwner          = "owner"
+	flagSubscriber     = "subscriber"
+	flagTzoffset       = "tzoffset"
+)
+
+// nolint: dupl
+func NewTxCmd() *cobra.Command {
+	subscriptionTxCmd := &cobra.Command{
+		Use:                        types.ModuleName,
+		Short:                      "Subscription transactions subcommands",
+		DisableFlagParsing:         true,
+		SuggestionsMinimumDistance: 2,
+		RunE:                       client.ValidateCmd,
+	}
+
+	subscriptionTxCmd.AddCommand(
+		NewCmdCreatePlan(),
+		NewCmdStopPlan(),
+		NewCmdCreateSubscription(),
+		NewCmdStopSubscription(),
+		NewCmdStopUserSubscription(),
+	)
+
+	return subscriptionTxCmd
+}
+
+// NewCmdCreatePlan implements submitting a create plan transaction command.
+func NewCmdCreatePlan() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create-plan",
+		Short: "Create a subscription plan",
+		Long:  "",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+			// Get voting address
```

Comment:
owner address instead of voting address?

Reply:
fixed